src/runners/scilla_merger.ml

```diff
@@ -0,0 +1,86 @@
+(*
+  This file is part of scilla.
+
+  Copyright (c) 2018 - present Zilliqa Research Pvt. Ltd.
+
+  scilla is free software: you can redistribute it and/or modify it under the
+  terms of the GNU General Public License as published by the Free Software
+  Foundation, either version 3 of the License, or (at your option) any later
+  version.
+
+  scilla is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+  A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License along with
+  scilla.  If not, see <http://www.gnu.org/licenses/>.
+*)
+
+open Core
+open Scilla_base
+open Scilla_product
+open Product
+open JSONProduct
+open ErrorUtils
+module MProduct = ScillaProduct (Checker.TCSRep) (Checker.TCERep)
+
+module Fmt =
+  Formatter.Format (Checker.TCSRep) (Checker.TCERep) (MProduct.PLiteral)
+
+let parse_args args =
```

Comment:
A suggestion: let's reuse Cmdliner for newer binaries: it would make it easier to extend the interface with more options. It also makes the binary nicely documented.

Reply:
It depends on the arguments parsing function implemented in the `RunnerUtil.ml`:  https://github.com/Zilliqa/scilla/blob/5e6f8f931dfb3bd5f5ab044beee2aefaa14662b3/src/base/RunnerUtil.ml#L209.

If we add `Cmdliner` here, we'll need to convert the arguments back to a list of strings in order to use the above function. So, I suggest to refactor this later when implementing `Cmdliner` interface for the `scilla-checker`.